Makefile

```diff
@@ -40,6 +40,10 @@ docker-build:
 offsets:
 	cd offsets-tracker; OFFSETS_OUTPUT_FILE="../pkg/inject/offset_results.json" go run main.go
 
+.PHONY: docker-offsets
+docker-offsets:
+	docker run --rm -v $(shell pwd):/app golang:1.20 /bin/sh -c "cd ../app && make offsets"
```

Comment:
This looks like it is going to be brittle given when a new version of Go comes out this will need to be manually bumped to allow the new generation, right?

Reply:
Yeah, but that's true for our other workflows that use go (and our dockerfiles/go.mod).

If we want one less place to have to update with new minor version releases, we could use `golang:latest` here. But then there will be skew if the `latest` image updates before we update our own builds. So pinning to the latest minor version we've tested on seems reasonable.